docs/handbook/chapter4/runtime-policy.md

```diff
@@ -0,0 +1,19 @@
+Runtime policy or Sealevel runtime account rules are a set of rules enforced by the Sealevel runtime to protect the security of the system and make Solana a safe and predictable environment for its users.
+
+**The following list of rules is taken from the official documentation:**
```

Comment:
Is this the same as  https://github.com/Ackee-Blockchain/school-of-solana/tree/master/6.lesson#solana-runtime-policy ?

Reply:
Yes.